Cannot find module 'gradient-parser'

When building a typescript project that rely on `pixi.js` and `pixi-filters`, I get the following error:

```sh
Error: node_modules/pixi-filters/lib/color-gradient/CssGradientParser.d.ts:1:122 - error TS2307: Cannot find module 'gradient-parser' or its corresponding type declarations.

1 import { AngularNode, ColorStop as CssColorStop, DefaultRadialNode, DirectionalNode, ExtentKeywordNode, ShapeNode } from 'gradient-parser';
```

I'm using:
- pixi.js `^8.1.0`
- pixi-filters `^6.0.0`

--- a/package.json
+++ b/package.json
@@ -222,11 +222,13 @@
   "eslintConfig": {
     "extends": "@pixi/eslint-config"
   },
+  "dependencies": {
+    "@types/gradient-parser": "^0.1.2"
+  },
   "devDependencies": {
     "@pixi/eslint-config": "^4.0.1",
     "@pixi/extension-scripts": "^2.4.0",
     "@pixi/webdoc-template": "^2.2.3",
-    "@types/gradient-parser": "^0.1.2",
     "@webdoc/cli": "^2.2.0",
     "base64-to-image": "^1.0.2",
     "copyfiles": "^2.4.1",

